src/docs/installation-docker.md

```diff
@@ -2,6 +2,111 @@
 title: Docker
 group: Installation
 ---
+> # Setup of Photoview in Docker
+
+
+> ## Setting up Docker Compose
+
+1. Editing Docker Compose 
+
+**IMPORTANT: DOCKER COMPOSE WILL AUTOMATICALLY SETUP MYSQL AND DB**
+
+**Things to Note:**
+**When setting up the volumes where your media will be read. All images/videos have to be in a folders root to show!**
+**So ie. **
+
+`      - ./mnt/Win10-4TB-HDD/Google-Photos/GPhotos/Sons-Videos`
+
+**All video that are going to be shown have to be in /Sons-videos for the volume to work correctly.**
+
+1. Open the Home directory
+
+`cd ~/`
+
+2. Edit docker-compose.yml using your editor of choice
+
+`sudo nano docker-compose.yml`
+
+3. Find the comments starting with Change This:, and change the values, to properly match your setup. If you are just testing locally, you don't have to change the
+
+```      
+      - ./home/demonwarrior/photos:/photos:ro
+      - ./mnt/Win10-4TB-HDD/Google-Photos/GPhotos/Sons-Videos
+```
+
+to the directories that have the images or videos that you would like to add from your system.
```

Comment:
A proper explaination of how docker bind volumes work would be nice to explain confusion for people unfamiliar with the Docker ecosystem